src/test/integration/mocks/notification-message.ts

```diff
@@ -38,9 +44,18 @@ export const assertInfoMessageDisplayed = async (message: string) => {
 };
 
 export const assertErrorMessageDisplayed = async (message: string) => {
-  // wait for setInterval to invoke Environments provider state refresh
-  await waitFor(() => expect(showErrorMessageSpy).toHaveBeenCalled());
-  expect(showErrorMessageSpy).toHaveBeenCalledWith(message, "More info");
+  await waitFor(() => {
+    // ignore the error messages related to getDefaultBranch
+    const calls = showErrorMessageSpy.mock.calls.filter(
+      (call) =>
+        !(
+          call[0].startsWith(cannotGetDefaultBranchMessage) ||
+          call[0].startsWith(getDefaultBranchErrorMessage)
+        )
+    );
```

Comment:
what errors? I don't understand this one

Reply:
Since the simple-git library is imported into the code before the test files load, making it impossible to mock, I am encountering error messages in the tests for getting the default branch. Therefore, I need to ignore the cannotGetDefaultBranchMessage and getDefaultBranchErrorMessage messages.

Reply:
Wdym by its loaded before the test file? how is it different from other libs?